src/schema/broadband-signup-au.json

```diff
@@ -141,28 +181,172 @@
+                            "enum": [
+                                "owned",
+                                "rent",
+                                "others"
+                            ]
+                        },
+                        "moveingInDate": {
+                            "type": "string",
+                            "format": "date",
+                            "example": "2001-01-01"
+                        }
+                    },
+                    "required": [
+                        "line1",
+                        "postCode",
+                        "city",
+                        "citySubdivision",
+                        "countrySubdivision",
+                        "status"
+                    ],
+                    "additionalProperties": false
+                },
+                "previousAddress": {
+                    "type": "object",
+                    "properties": {
+                        "line1": {
+                            "type": "string",
+                            "example": "50 CRESTHAVEN BVD",
+                            "minLength": 1
+                        },
+                        "line2": {
+                            "type": "string",
+                            "example": "Flat 2"
+                        },
+                        "postCode": {
+                            "type": "string",
+                            "example": "3806",
+                            "minLength": 1
+                        },
+                        "city": {
+                            "type": "string",
+                            "example": "Melbourne",
+                            "minLength": 1
+                        },
+                        "citySubdivision": {
+                            "type": "string",
+                            "example": "Berwick",
+                            "minLength": 1
+                        },
+                        "countrySubdivision": {
+                            "$ref": "#/domains/BroadbandSignupAU/types/StateCode"
+                        }
+                    },
+                    "required": [
+                        "line1",
+                        "postCode",
+                        "city",
+                        "citySubdivision",
+                        "countrySubdivision"
+                    ],
+                    "additionalProperties": false
+                }
+                   
+            },
+            "required": [
+                "currentAddress"
+            ],
+            "additionalProperties": false
+        },
+        "Employment": {
+            "type": "object",
+            "properties": {
+                "occupation": {
                     "type": "string",
-                    "example": "Min. total cost $960 over 12 month(s) + $15 P&H"
+                    "example": "Product Manager",
+                    "minLength": 1
+                },
+                "industry": {
```

Comment:
Is this standardised in Australia? Can it be an enum?

Reply:
We need to research that. For now will remain without enum.